src/client/main.c

```diff
@@ -63,7 +63,7 @@ const Method methods[] = {
         { "status",          0, ARG_ANY, OPT_WATCH,                               method_status,             usage_method_status             },
         { "set-loglevel",    1, 2,       OPT_NONE,                                method_set_loglevel,       usage_method_set_loglevel       },
         { "get-default",     1, 1,       OPT_NONE,                                method_get_default_target, usage_method_get_default_target },
-        { "set-default",     2, 3,       OPT_NONE,                                method_set_default_target, usage_method_set_default_target },
+        { "set-default",     2, 2,       OPT_FORCE,                               method_set_default_target, usage_method_set_default_target },
```

Comment:
We looked at systemctl and they don't provide option to pass force flag for set-default, force is enabled by default:

https://github.com/systemd/systemd/blob/main/src/systemctl/systemctl-set-default.c#L130

That's why we agredd with Artiom to copy that approach

Reply:
Makes sense. 
However, the previous implementation didn't reflect that since an optional force could've been passed, e.g. `bluechictl set-default <target> true`. Lets stick with the agreed default of passing `1` as in systemd's implementation for `bluechictl`. I'll update the PR shortly.

Reply:
Sorry, we missed the incorrect number of params in the code :-(

Reply:
No worries :) Updated PR and added a small inline-comment about the `1` as fixed value in the API. Could you have another look? @mwperina